{
  message: {
    function: 'AuthService.googleAuthVerify',
    error: TypeError: Cannot read properties of undefined (reading 'value')
        at Strategy.googleAuthVerify [as _verify] (/run/media/adrianabadin/code/informe5Backend/auth/auth.service.ts:78:40)
        at /run/media/adrianabadin/code/informe5Backend/node_modules/passport-oauth2/lib/strategy.js:198:24
        at /run/media/adrianabadin/code/informe5Backend/node_modules/passport-google-oauth2/lib/oauth2.js:165:7
        at passBackControl (/run/media/adrianabadin/code/informe5Backend/node_modules/oauth/lib/oauth2.js:134:9)
        at IncomingMessage.<anonymous> (/run/media/adrianabadin/code/informe5Backend/node_modules/oauth/lib/oauth2.js:157:7)
        at IncomingMessage.emit (node:events:525:35)
        at endReadableNT (node:internal/streams/readable:1359:12)
        at processTicksAndRejections (node:internal/process/task_queues:82:21)
  },
  level: 'error'
}
{
  message: {
    function: 'AuthService.googleAuthVerify',
    error: TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
        at Strategy.googleAuthVerify [as _verify] (/run/media/adrianabadin/code/informe5Backend/auth/auth.service.ts:78:25)
        at /run/media/adrianabadin/code/informe5Backend/node_modules/passport-oauth2/lib/strategy.js:198:24
        at /run/media/adrianabadin/code/informe5Backend/node_modules/passport-google-oauth2/lib/oauth2.js:165:7
        at passBackControl (/run/media/adrianabadin/code/informe5Backend/node_modules/oauth/lib/oauth2.js:134:9)
        at IncomingMessage.<anonymous> (/run/media/adrianabadin/code/informe5Backend/node_modules/oauth/lib/oauth2.js:157:7)
        at IncomingMessage.emit (node:events:525:35)
        at endReadableNT (node:internal/streams/readable:1359:12)
        at processTicksAndRejections (node:internal/process/task_queues:82:21)
  },
  level: 'error'
}
{ message: 'caboom', level: 'error' }
{ message: 'caboom', level: 'error' }
{ message: 'caboom', level: 'error' }
{ message: 'caboom', level: 'error' }
{ message: 'caboom', level: 'error' }
{ message: 'caboom', level: 'error' }
{ message: 'caboom', level: 'error' }
{
  message: {
    function: 'AuthService.googleAuthVerify',
    error: PrismaClientInitializationError: 
    Invalid `this.prisma.users.findUnique()` invocation in
    /run/media/adrianabadin/code/informe5Backend/auth/auth.service.ts:77:27
    
      74 public googleAuthVerify = (req: Request, accessToken: string, refreshToken: string, profile: any, done: DoneType) => {
      75   try {
      76     const { email } = profile
    → 77     this.prisma.users.findUnique(
    Can't reach database server at `127.0.0.1`:`3306`
    
    Please make sure your database server is running at `127.0.0.1`:`3306`.
        at pn.handleRequestError (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:6750)
        at pn.handleAndLogRequestError (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:5907)
        at /run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:3087
        at /run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:3294
        at t._executeRequest (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10755)
        at t._request (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10484) {
      clientVersion: '4.13.0',
      errorCode: undefined
    }
  },
  level: 'error'
}
{
  message: {
    function: 'AuthService.googleAuthVerify',
    error: PrismaClientInitializationError: 
    Invalid `this.prisma.users.findUnique()` invocation in
    /run/media/adrianabadin/code/informe5Backend/auth/auth.service.ts:77:27
    
      74 public googleAuthVerify = (req: Request, accessToken: string, refreshToken: string, profile: any, done: DoneType) => {
      75   try {
      76     const { email } = profile
    → 77     this.prisma.users.findUnique(
    Can't reach database server at `127.0.0.1`:`3306`
    
    Please make sure your database server is running at `127.0.0.1`:`3306`.
        at pn.handleRequestError (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:6750)
        at pn.handleAndLogRequestError (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:5907)
        at /run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:3087
        at /run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:3294
        at t._executeRequest (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10755)
        at t._request (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10484) {
      clientVersion: '4.13.0',
      errorCode: undefined
    }
  },
  level: 'error'
}
{
  message: {
    function: 'DatabaseHandler.create',
    error: PrismaClientValidationError: 
    Invalid `prisma.posts.create()` invocation:
    
    {
      data: {
        author: {
          connect: {
            id: 'usuario'
          }
        },
        classification: 'Municipales',
        heading: 'Wado presidente',
        title: 'Breaking News',
        text: 'Finaly he made it ',
        importance: '1',
                    ~~~
        images: {
          create: [
            {
              url: 'https://www.facebook.com/photo.php?fbid=709951107804705&set=p.709951107804705&type=3'
            },
            {
              url: 'https://www.facebook.com/photo.php?fbid=709951114471371&set=p.709951114471371&type=3'
            }
          ]
        }
      }
    }
    
    Argument importance: Got invalid value '1' on prisma.createOnePosts. Provided String, expected Int.
    
    
        at mi.validate (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:149:91)
        at tn.createMessage (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:166:1205)
        at /run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10962
        at runInChildSpan (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:70:25817)
        at t._executeRequest (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10951)
        at t._request (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10484) {
      clientVersion: '4.13.0'
    }
  },
  level: 'error'
}
{
  message: {
    function: 'DatabaseHandler.create',
    error: PrismaClientKnownRequestError: 
    Invalid `prisma.posts.create()` invocation:
    
    
    An operation failed because it depends on one or more records that were required but not found. No 'Users' record(s) (needed to inline the relation on 'Posts' record(s)) was found for a nested connect on one-to-many relation 'PostsToUsers'.
        at pn.handleRequestError (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:6477)
        at pn.handleAndLogRequestError (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:5907)
        at pn.request (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:176:5786)
        at t._request (/run/media/adrianabadin/code/informe5Backend/node_modules/@prisma/client/runtime/library.js:179:10484) {
      code: 'P2025',
      clientVersion: '4.13.0',
      meta: {
        cause: "No 'Users' record(s) (needed to inline the relation on 'Posts' record(s)) was found for a nested connect on one-to-many relation 'PostsToUsers'."
      }
    }
  },
  level: 'error'
}
